bittensor/_neuron/text/template_miner/__init__.py

```diff
@@ -92,7 +92,7 @@ def add_args( cls, parser: argparse.ArgumentParser ):
             # ---- Add neuron args.
             parser.add_argument('--config', type=str, help='If set, defaults are overridden by passed file.')
             parser.add_argument('--neuron.learning_rate', type=float, help='Training initial learning rate.', default=1)
-            parser.add_argument('--neuron.learning_rate_chain', type=float, help='Training initial learning rate.', default=1)
+            parser.add_argument('--neuron.learning_rate_chain', type=float, help='Training initial learning rate.', default=0.1)
```

Comment:
This has the exact same description as the regular learning rate above it. what's the difference?

Reply:
We don't actually use this anymore, let me remove it.